feat(livestock-animal-statistics): add animal statistics documentation (#33)
Pure-Farming/Documentation: docs/resource-types/livestock/animal-statistics.md

| ---------------------------- | ------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
| **Self**                     | A link to this specific animal statistic                                                                | URI                                                                                      |
| **Location**                 | An identifier for the location of the animal statistic                                                  | [Identifier](/resource-types/common.md#identifier)                                       |
| **Meta**                     | Meta data for the resource                                                                              | [Metadata](/resource-types/common.md#metadata)                                           |
| **Resource Type**            | The fixed discriminator for the animal statistic resource type.<br/>Value: /livestock/animal-statistics | String                                                                                   |

> | **Start Date**               | The time the statistic started                                                                          | Datetime                                                                                 |

Reviewer: ```suggestion
| **Start Date** | The start of the period covered by the statistics | Datetime |
```

> | **End Date**                 | The time the statistic ended                                                                            | Datetime                                                                                 |

Reviewer: ```suggestion
| **End Date** | The end of the period covered by the statistics | Datetime |
```

| **Duration**                 | Defines the duration that the statistics apply to                                                       | Enumeration                                                                              |
| **Id**                       | The Pure Farming Id of this animal statistic                                                            | UUID                                                                                     |
| **Inventory Classification** | Defines the group of animals by their characteristics to which the statistics refer                     | [Inventory Classification](/resource-types/livestock/common.md#inventory-classification) |

> | **Statistics**               | Holds animal statistics for the specified period for a group of livestock                               | Array of [Statistics](#statistics)                                                       |

Reviewer: ```suggestion
| **Statistics** | Statistics that apply to the group of animals in this period | Array of [Statistic](#statistic)s |
```

---

> ### Statistics

Reviewer: ```suggestion
### Statistic
```

> Holds animal statistics for the specified period for a group of livestock

Reviewer: ```suggestion
A statistic on a group of animals for the specified period
```

```json

> [{
>   "metric": {...},
>   "qualifier": {...},
>   "units": "string",
>   "value": 12,
>   "lastKnownValue": 2,
>   "lastKnownValueDate": "2022-08-01",
>   "aggregation": "string"
> }]

Reviewer: ```suggestion
{
  "metric": {
    "scheme": "antibioticUsage",
    "id": "udder"
  },
  "qualifier": {
    "scheme": "eu.ema",
    "id": "DDDVet"
  },
  "units": "DDDVet",
  "value": 12,
  "lastKnownValue": 2,
  "lastKnownValueDate": "2022-08-01",
  "aggregation": "COUNT"
}
```
